registry/remote/example_test.go

```diff
@@ -352,25 +400,107 @@ func ExampleRepository_Fetch_manifestByDigest() {
 	}
 	// verify the fetched content
 	if descriptor.Size != int64(len(pulled)) || descriptor.Digest != digest.FromBytes(pulled) {
-		panic(err)
+		panic("wrong content")
 	}
 
 	fmt.Println(string(pulled))
 	// Output:
 	// Example manifest content
 }
 
-// ExampleRepository_FetchReference_manifestByTag gives example snippets for downloading a manifest by tag with only one API call.
-func ExampleRepository_FetchReference_manifestByTag() {
-	reg, err := remote.NewRegistry(host)
+// ExampleRepository_Fetch_referenceManifestByDescriptor gives an example of fetching
+// an reference manifest by its descriptor.
+func ExampleRepository_Fetch_referenceManifestByDescriptor() {
```

Comment:
Wo don't need this example, as this is not a valid use case.

Reply:
Deleted this example.